protocol/tags.go

```diff
@@ -73,7 +73,7 @@ const PingReplyTagMaxSize = 8
 
 // ProposalPayloadTagMaxSize is the maximum size of a ProposalPayloadTag message
 // This value is dominated by the MaxTxnBytesPerBlock
-const ProposalPayloadTagMaxSize = 5250301
+const ProposalPayloadTagMaxSize = 5250313
```

Comment:
how is this calculated?

Reply:
There's a unit test that yells when you add a field, and tells you the new number.